source/state_representation/src/space/cartesian/CartesianState.cpp

```diff
@@ -69,6 +69,17 @@ CartesianState CartesianState::operator*(double lambda) const {
   return result;
 }
 
+CartesianState& CartesianState::operator/=(double lambda) {
+  lambda = 1.0 / lambda;
```

Comment:
this will need to throw an exception in case `lambda == 0`. You would need actually `abs(lambda) < eps` with eps sufficiently small

Reply:
good catch thanks